express-post-json/server.js

```diff
@@ -0,0 +1,29 @@
+import express from './node_modules/express/index.js';
+
+const app = express();
+
+let nextId = 1;
+const grades = {}
+
+app.get('/api/grades', (req, res) => {
+  const newArray = [];
+  for (let keys in grades) {
+    newArray.push(grades[keys]);
+  }
+  res.json(newArray);
+});
+
+app.use(express.json());
```

Comment:
Usually we'd put this at the top since it can be used by all routes. It's only critical that this appear before routes that expect JSON, but it is safe to use on all routes.